### DISCUSSION TITLE
Decrease in contribution count

### DISCUSSION THREAD

**Discussion by thesanchitadevi** [community/community#67729] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am continuously losing my contributions to graphs. Yesterday, I saw it was 983. I just checked it, and it's now 964. Why is it happening?

**Comment by alexisLefebvre** [comment, marked as answer]
What are “graphs”? Where do you see these numbers?

**Reply by thesanchitadevi** [reply]
in the contributions graph

**Reply by alexisLefebvre** [reply]
The number counts contributions **in the last year**, so your contributions older than 1 year are not counted anymore, and the total number is lower as time pass. If you stop contributing during one full year, it will be 0.
